localization/ReleaseNotes.txt

```diff
@@ -1,3 +1,13 @@
+v1.7.2
+
+* Fix performance issues on login
+* Fix Split BCH/BSV Error
+* Various visual and text updates
+* Add tap address to launch explorer at request or send confirmation screens
+* Add restore wallet using private seed functionality
```

Comment:
We should remove the "restore wallet" feature, since we didn't actually activate it.